vnext/PropertySheets/ReactDirectories.props

```diff
@@ -20,20 +25,11 @@
     <OutDir>$(BaseOutDir)\$(ProjectName)\</OutDir>
 
     <GeneratedFilesDir>$(IntDir)Generated Files\</GeneratedFilesDir>
-
     <ReactNativeDir Condition="'$(ReactNativeDir)' == ''">$(IntDir)\react-native-patched\</ReactNativeDir>
 
-    <!-- For relative paths, make it relative to MSBuildProjectDirectory not CurrentWorkingDirectory. -->
-    <ReactNativeDir>$([MSBuild]::NormalizeDirectory( $([System.IO.Path]::Combine( $(MSBuildProjectDirectory), $(ReactNativeDir) )) ))</ReactNativeDir>
```

Comment:
This should no longer be needed since we now normalize ReactNativeWindowsDir earlier on, meaning IntDir is always normalized (apart from extra slashes, but not traversals).